nwb_conversion_tools/datainterfaces/ecephys/axona/axonadatainterface.py

```diff
@@ -135,9 +136,10 @@ def get_source_schema(cls):
             type="object",
         )
 
-    def __init__(self, filename: FilePathType, noise_std: float = 3.5):
-        super().__init__(filename=filename)
-        self.recording_extractor = se.AxonaUnitRecordingExtractor(filename=filename, noise_std=noise_std)
+    def __init__(self, file_path: FilePathType, noise_std: float = 3.5):
+        super().__init__(filename=file_path)
+        self.source_data = dict(file_path=file_path)
+        self.recording_extractor = se.AxonaUnitRecordingExtractor(filename=file_path, noise_std=noise_std)
 
```

Comment:
```suggestion
super().__init__(file_path=file_path)
```

Reply:
However, looking at this particular subinterface and the way it inherits, it wouldn't work either way regardless. Added a debug for that.

Though, I don't think this interface is tested in any way so it is difficult to tell.